asterisk/rootfs/etc/services.d/asterisk_mbox/run

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/with-contenv bashio
+# ==============================================================================
+# Start the asterisk mailbox service
+# s6-overlay docs: https://github.com/just-containers/s6-overlay
+# ==============================================================================
+
+# shellcheck shell=bash
+
+if bashio::var.true "$(bashio::config 'mailbox')"; then
+    bashio::log.info "Starting Asterisk Mailbox"
+
+    ## Run Asterisk_mbox
+    exec /usr/local/bin/asterisk-mbox-server --cfg /etc/asterisk/asterisk_mbox.ini -U asterisk
+fi
```

Comment:
I suggest a different approach, how about in `cont-init.d` you do:

```bash
if bashio::var.false "$(bashio::config 'mailbox')"; then
  rm -rf /etc/services.d/asterisk_mbox
fi
```

Reply:
It checks if it should start in the service itself.

```
if bashio::var.true "$(bashio::config 'mailbox')"; then
    bashio::log.info "Starting Asterisk Mailbox"

    ## Run Asterisk_mbox
    exec /usr/local/bin/asterisk-mbox-server --cfg /etc/asterisk/asterisk_mbox.ini -U asterisk
fi
```

Tested it, works fine and don't see anything about it in the console either.

Reply:
But your approach should work too. Feel free to comment in https://github.com/just-containers/s6-overlay/issues/394#issuecomment-1028432171 to see what they think about it.